lib/exception_notifier.rb

```diff
@@ -104,7 +104,9 @@ def ignored?(exception, options)
     end
 
     def ignored_exception?(ignore_array, exception)
-      (Array(ignored_exceptions) + Array(ignore_array)).map(&:to_s).include?(exception.class.name)
+      all_ignored_exceptions = (Array(ignored_exceptions) + Array(ignore_array)).map(&:to_s)
+      exception_ancestors = exception.class.ancestors.map(&:to_s)
+      !(all_ignored_exceptions & exception_ancestors).empty?
```

Comment:
This is using the intersection operator (see https://stackoverflow.com/questions/3941945/array-include-any-value-from-another-array).